openmc/deplete/__init__.py

```diff
@@ -0,0 +1,24 @@
+"""
+openmc.deplete
+==============
+
+A depletion front-end tool.
+"""
+
+from .dummy_comm import DummyCommunicator
+try:
+    from mpi4py import MPI
+    comm = MPI.COMM_WORLD
+    have_mpi = True
+except ImportError:
+    comm = DummyCommunicator()
+    have_mpi = False
```

Comment:
Can we make `comm` a member of the `Operator` class?  Furthermore, it would be nice if the user can pass a communicator that's more specific than COMM_WORLD.

Reply:
`comm` actually used to be a member of `Operator` and I had moved it to a module-level variable because it is used as part of other classes (e.g., `Results`). We could have it be part of `Operator` again, but it would require passing it around. I would propose just keeping it as a module-level variable and adding some documentation mentioning it. It can be changed by a user, e.g.
```Python
openmc.deplete.comm = my_intracomm
```

Reply:
Sounds good to me.